main.tf

```diff
@@ -25,8 +25,8 @@ data "google_project" "project" {
 }
 
 locals {
+  default_subscription_label   = { "managed_by" : "terraform" }
```

Comment:
Doesn't look like this is being used any where but if we can merge it with the subscription and topic labels we could automatically make sure this gets added.